.swm/JFXftJml8DpmuCPBA9rL.swm

```diff
@@ -0,0 +1,30 @@
+            ]
+        }
+    },
+    "app_version": "0.1.90",
+    "file_version": "1.0.2"
+}
```

Comment:
Newline?

Reply:
Ah, Swimm committed these files. I reckon we should let these be just for consistency? Otherwise we'll have to edit the Swimm commit and add a newline every time.